File: batch11amsm/FirstApplication/app/src/main/java/com/example/firstapplication/fragments/SecondFragment.java
package com.example.firstapplication.fragments;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.example.firstapplication.R;


public class SecondFragment extends Fragment {

TextView seondtext;
String emailvalue;
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_second, container, false);
    }

    @Override
    public void onViewCreated( View view,  Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        seondtext = view.findViewById(R.id.secondtextivew);
        emailvalue = getArguments().getString("data");

        seondtext.setText(emailvalue);
    }
}

File: kushalapp/BasickComponent/app/src/main/java/com/cbitss/basickcomponent/fragments/SecondFragment.kt
package com.cbitss.basickcomponent.fragments

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import com.cbitss.basickcomponent.R

class SecondFragment : Fragment() {
lateinit var textview:TextView

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_second, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        textview = view.findViewById(R.id.textviewsecond)
        if(arguments != null)
        {
           var emailvalue = requireArguments().getString("myemail")
            textview.text = emailvalue
        }
    }
}